docs/components/ColorsView.jsx

```diff
@@ -1,14 +1,263 @@
 import React, {PureComponent} from "react";
 
+import Example from "./Example";
 import View from "./View";
+import {Grid} from "src";
+
+import "./ColorsView.less";
 
 export default class ColorsView extends PureComponent {
   render() {
     const {cssClass} = ColorsView;
-
+    const {Col, Row} = Grid;
     return (
       <View className={cssClass.CONTAINER} title="Colors">
-        TODO
+        <Grid>
+          <Row grow className="items--stretch margin--bottom--5xl padding--right--m">
+            <Col span={6} className="flexbox self--start padding--right--l">
+              <div className="display--block">
```

Comment:
can drop `display--block`. since `<div>`s are `display: block` by default, this ends up being bloat.
would be useful as an override for `<a>`s and `<span>`s